crates/hdi/CHANGELOG.md

```diff
@@ -7,21 +7,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
-## 0.3.1-beta-dev.0
```

Comment:
all of these should be kept.

develop [is already on the 0.4 series](https://github.com/holochain/holochain/blob/develop/crates/hdi/CHANGELOG.md#030)